### ISSUE TITLE
[Screen Reader-Social Embedded-Edit Profile]: Label should be defined for  "Toggle" button under privacy.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#12]
"Priority: High

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Fill all the input fields with valid inputs.
10. Navigate to ""Done"" button and press ENTER.
11. Navigate to ""Open Navigation Drawer"" button and press ENTER  (or double tap).
12. Navigate to ""My Profile"" and press ENTER  (or double tap) to open.
13. Navigate to  ""Edit Profile"" and press ENTER  (or double tap) to open.
14. Navigate to  ""Toggle"" button under ""Privacy"" .
15. Verify that label is defined for ""Toggle"" button or not.

Expected
Label should be defined for  ""Toggle"" button under privacy, So that Talk Back should announce ""Only approved followers can see my posts & comments toggle button switch ON"" or   ""Only approved followers can see my posts & comments toggle button switch OFF"".

Actual
Label is not  defined for  ""Toggle"" button under privacy, Talk Back is announcing only ""On Switch"" and ""Off Switch"". 

MAS Rule: MAS4.2.5

Impact
User who is rely on screen reader will not get complete information and might get confuse if label is not defined for the ""Toggle"" button.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back"
